maud_macros/src/parse.rs

```diff
@@ -553,6 +553,7 @@ impl Parser {
                     );
                 }
             },
+            None => abort_call_site!("expected ';' or '/', found end of macro"),
```

Comment:
I'd rather only mention `;`, as the `/` syntax is soft-deprecated.

Also, Rust style is to use backticks to delimit code snippets.

```suggestion
            None => abort_call_site!("expected `;`, found end of macro"),
```